draft-ietf-tls-tls13.md

```diff
@@ -916,129 +916,11 @@ itself is not protected by encryption unless the sender uses the
 supplied padding mechanism -- see {{record-padding}} for more details.
 
 
-##  Connection States
-
-[[TODO: I plan to totally rewrite or remove this. IT seems like just cruft.]]
-
-A TLS connection state is the operating environment of the TLS Record
-Protocol.  It specifies a record protection algorithm and its
-parameters as well as the record protection keys and IVs for the
-connection in both the read and the write directions. The security
-parameters are set by the TLS Handshake Protocol, which also determines
-when new cryptographic keys are installed and used for record
-protection.
-The initial current state always specifies that records are
-not protected.
-
-The security parameters for a TLS Connection read and write state are set by
-providing the following values:
-
-{:br: vspace="0"}
-
-connection end
-
-: Whether this entity is considered the "client" or the "server" in
-  this connection.
-
-Hash algorithm
-
-: An algorithm used to generate keys from the appropriate secret (see
-  {{key-schedule}} and {{traffic-key-calculation}}).
-
-record protection algorithm
-
-: The algorithm to be used for record protection. This algorithm must
-  be of the AEAD type and thus provides integrity and confidentiality
-  as a single primitive. This specification
-  includes the key size of this algorithm and of the nonce for
-  the AEAD algorithm.
-
-master secret
-
-: A 48-byte secret shared between the two peers in the connection
-  and used to generate keys for protecting data.
-
-
-client random
-
-: A 32-byte value provided by the client.
-
-server random
-
-: A 32-byte value provided by the server.
-{: br}
-
-These parameters are defined in the presentation language as:
-
-%%% Security Parameters
```

Comment:
This fails to build:
https://travis-ci.org/tlswg/tls13-spec/builds/134950214

Looks like you also need to delete the section in the appendix that transcludes this. (currently section A.5)
https://tools.ietf.org/html/draft-ietf-tls-tls13-13#appendix-A.5
https://tlswg.github.io/tls13-spec/#the-security-parameters